runtime/oti/xhelpers.m4

```diff
@@ -1,4 +1,4 @@
-dnl Copyright (c) 1991, 2022 IBM Corp. and others
+dnl Copyright (c) 1991, 2021 IBM Corp. and others
```

Comment:
This copyright shouldn't be reverted to 2021, the file is modified by other commits in 2022.